tests/EntityTest.php

```diff
@@ -23,6 +23,21 @@ public function testContentCastsToDatatype()
 		$this->assertSame(12, $setting->content);
 	}
 
+	public function testContentJsonCastsToDatatype()
+	{
+		$array = ['a' => 'Bananas', 'b' => 'Oranges'];
+
+		$setting = new Setting([
+			'datatype' => 'json-array',
+			'content'  => $array,
+		]);
+
+		$check = $this->getPrivateProperty($setting, 'attributes')['content'];
+		$this->assertEquals('{"a":"Bananas","b":"Oranges"}', $check);
+
+		$this->assertEquals($array, $setting->content);
```

Comment:
Please use the tightest assertion possible:
```suggestion
		$this->assertSame($array, $setting->content);
```

Reply:
Good to know that there is a difference between those 😆